lib/src/package_meta.dart

```diff
@@ -82,6 +82,9 @@ abstract class PackageMeta {
   /// Use this instead of fromDir where possible.
   factory PackageMeta.fromElement(
       LibraryElement libraryElement, DartdocOptionContext config) {
+    // [config] is only here for sdkDir, and it's OK that it is the wrong
+    // context since sdkDir is argOnly and this is supposed to be a temporary
+    // workaround.
     // Workaround for dart-lang/sdk#32707.  Replace with isInSdk once that works.
```

Comment:
It looks like https://github.com/dart-lang/sdk/issues/32707 is closed.  Can this be updated to use `isInSdk`?